prow/git/testing/git_test.go

```diff
@@ -14,15 +14,17 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package git_test
+package testing
```

Comment:
Document why this is not `package git`

Reply:
Let's just leave a comment saying that this needs to be in a separate package so we don't introduce an import cycle.